org.eclipse.scout.rt/pom.xml

```
@@ -131,8 +131,9 @@
     <artemis.version>2.26.0</artemis.version>
     <com.google.http-client.version>1.42.3</com.google.http-client.version>
     <org.apache.httpcomponents.version>4.5.13</org.apache.httpcomponents.version>
-    <opentelemetry.version>1.34.1</opentelemetry.version>
-    <opentelemetry.instrumentation.version>2.0.0-alpha</opentelemetry.instrumentation.version>
+    <opentelemetry.version>1.44.1</opentelemetry.version>
+    <opentelemetry.instrumentation.version>2.10.0</opentelemetry.instrumentation.version>
+    <opentelemetry.instrumentation.alpha.version>${opentelemetry.instrumentation.version}-alpha</opentelemetry.instrumentation.alpha.version>
```

I would prefer duplicating the `instrumentation version` value here. It might be simpler/more clear. And the duplication is quite local (one line above), which reduces possible errors when updating the version(s).

```suggestion
    <opentelemetry.instrumentation.alpha.version>2.10.0-alpha</opentelemetry.instrumentation.alpha.version>
```